Dockerfile

```diff
@@ -20,7 +20,7 @@ ARG	GRAFANA_ENV
 
 COPY	./grafana/${GRAFANA_ENV}/	/grafana
 
-FROM	grafana/grafana-oss:12.1.0	AS	grafana
+FROM	grafana/grafana-oss:12.1.1	AS	grafana
```

Comment:
hopefully solving the issue with creating new alerts...
https://github.com/grafana/grafana/releases/tag/v12.1.1